### PR TITLE
Add nbrmgr to supervisor control 

### PR DESCRIPTION
**Pull request by prsunny**
**- What I did**
Add nbrmgr daemon to supervisor control 

**- How I did it**
N/A

**- How to verify it**
Check process running status

**- Description for the changelog**
Depends on https://github.com/Azure/sonic-swss/pull/693

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by prsunny** [comment]
retest this please

File: dockers/docker-orchagent/supervisord.conf [hunk 1]
```diff
@@ -117,6 +117,14 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:nbrmgrd]
+command=/usr/bin/nbrmgrd
+priority=15
```
Comment (jleveque) [inline dockers/docker-orchagent/supervisord.conf]: Duplicate priority with restore_neighbors. This shouldn't cause any issues with the way we use supervisor, but I thought I'd bring it to your attention to see if it was intentional. The same applies to the docker-sonic-vs file below.

Comment (prsunny) [reply dockers/docker-orchagent/supervisord.conf]: It was not intentional, fixed the values. Thanks for pointing out!

**Comment by prsunny** [comment]
**Submodule update sonic-swss-common:**

edbfeec - Remove default docker name value of swss. (#250)
9728462 - Corrected configDB name for neigh table (#251)
6decc65 - Add NEIGH_TABLE to configDB for neighbor configuration (#249)
9918ae6 - Add ProducerStateTable temp view implementation and UT (#247)
41408f2 - Update README on dependencies
d9c0ba4 -Update README on the section 'Build with Google Test'
bb7fa5b - [ut]: explicit convert is to bool type (#248)
661b82c - Add gtest instruction in README

**Submodule update sonic-swss**

705b092 - Support ConfigDB neighbor configuration, introduce nbrmgr daemon (#693)
8522390 - Add vxlan switch attributes to switch orch (#712)
b123fa0 - [schema] update WARM_RESTART_TABLE:process_name schema document (#707)
2d7ab0c - Revert "Align default MTU value as SAI default (#705)" (#710)
836a58c - Align default MTU value as SAI default (#705)
bffa01f - VNET/VXLAN changes (#643)
b750a4b - [watermarkorch] add watermarkorch, extend queue and pg counters with wat\u2026 (#629)

**Pull request merged**